python/foglamp/services/core/api/task.py

```diff
@@ -180,6 +180,21 @@
         storage = connect.get_storage_async()
         config_mgr = ConfigurationManager(storage)
 
+        # Abort the operation if there are already executed tasks
+        payload = PayloadBuilder() \
+            .SELECT(["id", "schedule_name"]) \
+            .WHERE(['schedule_name', '=', name]) \
+            .LIMIT(1) \
+            .payload()
+
+        result = await storage.query_tbl_with_payload('tasks', payload)
+
+        if result['count'] >= 1:
+            msg = 'Invalid name {0}, there are tasks already executed with this name.'.format(name)
```

Comment:
I don't think this message is very meaningful to the user. Can I suggest something along the lines of "Unable to reuse name {}, already used by a previous task."

Reply:
thanks for the piece of advice, fixed